### DISCUSSION TITLE
xDrip now offers sensor expiry notifications

### DISCUSSION THREAD

**Discussion by Navid200** [NightscoutFoundation/xDrip#2670] [Show and tell]
It is added to today's Nightly: https://github.com/NightscoutFoundation/xDrip/releases/tag/2023.02.11

It gives you notifications 24 hours, 9 hours and 2 hours before expiry.
You can enable it under Extra Alerts (xDrip+).  https://navid200.github.io/xDrip/docs/Alerts_page.html

**Comment by steve8x8** [comment]
I had expected this to show up for translation on Crowdin...?

**Reply by Navid200** [reply]
Thanks
I'm opening a PR to add it.

**Reply by Navid200** [reply]
https://github.com/NightscoutFoundation/xDrip/pull/2677
